Gemfile

```diff
@@ -69,6 +69,10 @@ group :test do
   end
 end
 
+group :development, :test do
+  gem 'pry'
+end
+
```

Comment:
This does not really belong here.

Reply:
I would like to keep it around as long as needed, will remove it before the merge.